jenkins/parser/jenkins-parser-job.py

```diff
@@ -23,10 +23,22 @@ def grep(filename, pattern, verbose=False):
 def get_errors_list(jobs_object, job_id):
     """Get list of errors to check for a concrete job with the corresponding action."""
 
-    # Get the error keys of the concrete job ii
+    # Get errorMsg object
     jenkins_errors = jobs_object["jobsConfig"]["errorMsg"]
-
+    # Get common error messages and regex that must be force retried
+    common_keys = []
+    force_retry_regex = []
+    for ii in jobs_object["jobsConfig"]["errorMsg"].keys():
+        if jobs_object["jobsConfig"]["errorMsg"][ii]["allJobs"] == "true":
+            common_keys.append(ii)
+        if jobs_object["jobsConfig"]["errorMsg"][ii]["forceRetry"] == "true":
+            force_retry_regex.extend(
+                jobs_object["jobsConfig"]["errorMsg"][ii]["errorStr"]
+            )
+    # Get the error keys of the concrete job ii
     error_keys = jobs_object["jobsConfig"]["jenkinsJobs"][job_id]["errorType"]
+    error_keys.extend(common_keys)
```

Comment:
@aandvalenzuela , shouldn't we make a copy of `jobs_object["jobsConfig"]["jenkinsJobs"][job_id]["errorType"]` in above line? e.g. 
```
error_keys = jobs_object["jobsConfig"]["jenkinsJobs"][job_id]["errorType"][:]
```
I think otherwise `error_keys.extend(common_keys)` is going to add the new commonkeys in the original `job_object`